src/GravityFormsExtensions.php

```diff
@@ -278,11 +278,11 @@ function ( $conf ) use ( $confirmation ) {
 		$confirmation_fields = [
 			'confirmation'    => $default_confirmation['message'],
 			'share_platforms' => [
-				'facebook' => '1' === $default_confirmation['facebook'] ?? false,
-				'twitter'  => '1' === $default_confirmation['twitter'] ?? false,
-				'whatsapp' => '1' === $default_confirmation['whatsapp'] ?? false,
-				'native'   => '1' === $default_confirmation['native'] ?? false,
-				'email'    => '1' === $default_confirmation['email'] ?? false,
+				'facebook' => '1' === isset( $default_confirmation['facebook'] ) ? $default_confirmation['facebook'] : true,
+				'twitter'  => '1' === isset( $default_confirmation['twitter'] ) ? $default_confirmation['twitter'] : true,
+				'whatsapp' => '1' === isset( $default_confirmation['whatsapp'] ) ? $default_confirmation['whatsapp'] : true,
+				'native'   => '1' === isset( $default_confirmation['native'] ) ? $default_confirmation['native'] : true,
+				'email'    => '1' === isset( $default_confirmation['email'] ) ? $default_confirmation['email'] : true,
```

Comment:
Damm!!! 😬 I under look this condition because of  Null Coalescing Operator(??) operator usage, but because of 2 expression next to each other and as per Operator Precedence the first expression evaluated first (`'1' === $default_confirmation['email']`) which of-course produces undefined variable warning.

Here we can also simplifies it as -
`'email'    => $default_confirmation['email'] ?? true`